fulp_modules/main_features/bloodsuckers/code/powers/trespass.dm

```diff
@@ -12,9 +12,8 @@
 	var/turf/target_turf // We need to decide where we're going based on where we clicked. It's not actually the tile we clicked.
 
 /datum/action/bloodsucker/targeted/trespass/CheckCanUse(display_error)
-	. = ..()
-	if(!.)
-		return
+	if(!..())
```

Comment:
This change feels unnecessary, it makes the code less readable for a very negligible benefit. I think it'd be better if this part was reverted 
```suggestion
	. = ..()
	if(!.)
```

Reply:
All other powers do this so I can changed it for consistency

Reply:
Id prefer if it was consistently . = ..(), but it's not a huge deal either way